oneview/resource_server_profile_template.go

```diff
@@ -1541,12 +1533,11 @@ func resourceServerProfileTemplateCreate(d *schema.ResourceData, meta interface{
 		for _, raw := range rawFirmware {
 			firmwareItem := raw.(map[string]interface{})
 			firmware = ov.FirmwareOption{
-				ForceInstallFirmware:     firmwareItem["force_install_firmware"].(bool),
-				FirmwareBaselineUri:      utils.NewNstring(firmwareItem["firmware_baseline_uri"].(string)),
-				ManageFirmware:           firmwareItem["manage_firmware"].(bool),
-				FirmwareInstallType:      firmwareItem["firmware_install_type"].(string),
-				FirmwareActivationType:   firmwareItem["firmware_activation_type"].(string),
-				FirmwareScheduleDateTime: firmwareItem["firmware_schedule_date_time"].(string),
+				ForceInstallFirmware:   firmwareItem["force_install_firmware"].(bool),
+				FirmwareBaselineUri:    utils.NewNstring(firmwareItem["firmware_baseline_uri"].(string)),
+				ManageFirmware:         firmwareItem["manage_firmware"].(bool),
+				FirmwareInstallType:    firmwareItem["firmware_install_type"].(string),
+				FirmwareActivationType: firmwareItem["firmware_activation_type"].(string),
 			}
```

Comment:
Have we verified the case where profile is created from template and template contains firmware setting..what happen to those properties which are not exist in template but there in sp...hope it does not create any idempotency issues

Reply:
Yes, I have verified them. When attribute values change from SPT to SP, we dont get any idempotency issues in TF where as you can see the warning in OV saying SP is inconsistent with SPT.